spec/models/publish_services_event_metadata_spec.rb

```diff
@@ -0,0 +1,36 @@
+require 'rails_helper'
+
+RSpec.describe PublishServicesMetadataEvent, type: :model do
+  publish_date = Time.now
+  event_id = 0
+  component = Component.create(name: 'lala', component_type: 'MSA')
```

Comment:
These 3 lines should probably inside the let block or defined in their own let statements